### PR TITLE
Add the "processor_type" field to he QasmBackendConfiguration

### PR DESCRIPTION
**Pull request by taalexander**
### Summary
This PR adds the `processor_type` field to the `QasmBackendConfiguration`. This field has also been added to the schemas in https://github.com/Qiskit/ibm-quantum-schemas/pull/6

### CODE REVIEW

File: releasenotes/notes/add-config-processor-type-field-916f98f2eebf7e72.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Added the field ``processor_type`` to the
+    :class:`qiskit.providers.modelsQasmBackendConfiguration`. This contains information
```
Comment (jyu00) [inline releasenotes/notes/add-config-processor-type-field-916f98f2eebf7e72.yaml:5]:
```suggestion
    :class:`qiskit.providers.models.QasmBackendConfiguration`. This contains information
```

**Review by jyu00 — COMMENTED** [review]
missing dot in the release note, otherwise looks good

**Comment by nonhermitian** [comment]
What does segment A correspond to?  There is no labelling anywhere that indicates the subset of qubits to which this label is attached.

**Comment by taalexander** [comment]
> What does segment A correspond to? There is no labelling anywhere that indicates the subset of qubits to which this label is attached.

It is within the larger chip / fridge. This segment is for all qubits in the backend.

**Comment by nonhermitian** [comment]
Yes I know. But the label 'A' means nothing to an user. Where is the text that tells me what qubits 'A' corresponds to on the full topology?

**Review by nonhermitian — APPROVED** [review]

**Review by jyu00 — APPROVED** [review]

**Review by 1ucian0 — APPROVED** [review]

**Pull request merged**